app/views/support/organisations/claims/list.njk

```diff
@@ -17,6 +17,11 @@
   {% include "_includes/notification-banner.njk" %}
 
   <div class="govuk-grid-row">
+
+      {% block deadlineBanner %}
+        {% include "_includes/deadline-banner.njk" %}
+      {% endblock %}
+
     <div class="govuk-grid-column-two-thirds">
```

Comment:
I'm not sure you need the claim banner in the support site as they typically won't be creating claims.

```suggestion
  <div class="govuk-grid-row">
    <div class="govuk-grid-column-full">
      {% include "_includes/deadline-banner.njk" %}
    </div>
  </div>

  <div class="govuk-grid-row">
    <div class="govuk-grid-column-full">
```